Beam/app/Article.php

```diff
@@ -104,13 +108,17 @@ public function getVariantsCountAttribute(): array
             if (!$result->count) {
                 continue;
             }
-            $titleVariant = $result->tags->title_variant;
-            $imageVariant = $result->tags->image_variant;
-
-            if (!array_key_exists($titleVariant, $variants)) {
-                $variants[$titleVariant] = [];
+            if ($result->tags) {
+                $titleVariant = $result->tags->title_variant;
+                $imageVariant = $result->tags->image_variant;
+                if (!array_key_exists($titleVariant, $variants)) {
+                    $variants[$titleVariant] = [];
+                }
+                $variants[$titleVariant][$imageVariant] = $result->count;
+            } else {
+                $variants[DEFAULT_TITLE_VARIANT][DEFAULT_IMAGE_VARIANT] = $result->count;
```

Comment:
I don't think that defaults have any meaning here. If there are no values, there was no A/B test and we don't need to provide "single-variant" data for frontend. Single variant data are already displayed by frontend via the pageviews chart - the one with different color for each traffic source type.

Reply:
Based on what was written above, the defaults now make sense.